app/Commands/Dev/Core.hs

```diff
@@ -11,8 +11,9 @@ newtype CoreReplOptions = CoreReplOptions
   { _coreReplShowDeBruijn :: Bool
   }
 
-newtype CoreEvalOptions = CoreEvalOptions
-  { _coreEvalNoIO :: Bool
+data CoreEvalOptions = CoreEvalOptions
+  { _coreEvalShowDeBruijn :: Bool,
+    _coreEvalNoIO :: Bool
```

Comment:
The changes in this file are not related to this PR. In the future, please separate them in a different PR.

Reply:
Yes, you're right. I should've made a small PR to add `--show-de-bruijn` to `core eval`.

Reply:
Since I'm changing things, it won't be that much of a hassle to separate it into another PR. I'll try doing this.

Reply:
Done